SpatialGDK/Source/SpatialGDK/Public/Tests/SpatialView/TestWorker.h

```diff
@@ -11,16 +11,16 @@ namespace SpatialGDK
 class FTestWorker
 {
 public:
-	explicit FTestWorker(TArray<Worker_ComponentSetId> ComponentSetIds, FString InWorkerId = {}, Worker_EntityId InWorkerSystemEntityId = 0)
+	static FTestWorker Create(TArray<Worker_ComponentSetId> ComponentSetIds, FString InWorkerId = {},
```

Comment:
What was the reasoning behind this change?

I ask since the parameters for this "factory" method are the same as the original constructor?

Reply:
The issue was that the code didn't compile. The only `ViewCoordinator`'s constructor requires a `TUniquePtr<AbstractConnectionHandler>` which isn't exposed, and this class also requires a raw pointer to the same object. This static factory method bypasses this limitation.